usecases/schema/executor.go

```diff
@@ -54,15 +57,35 @@ func (e *executor) Open(ctx context.Context) error {
 func (e *executor) ReloadLocalDB(ctx context.Context, all []api.UpdateClassRequest) error {
 	cs := make([]*models.Class, len(all))
 
+	g, ctx := enterrors.NewErrorGroupWithContextWrapper(e.logger, ctx)
+	g.SetLimit(runtime.GOMAXPROCS(0) * 2)
```

Comment:
can we define `_NUMCPU` variable in this package like so:
```
var _NUMCPU = runtime.GOMAXPROCS(0)
```
and then use it here and in the loop over `e.callbacks`?